.github/workflows/conda-python-build.yaml

```diff
@@ -42,56 +48,18 @@ permissions:
     statuses: none
 
 jobs:
-    compute-matrix:
-      runs-on: ubuntu-latest
-      outputs:
-        MATRIX: ${{ steps.compute-matrix.outputs.MATRIX }}
-      steps:
-        - name: Compute Build Matrix
-          id: compute-matrix
-          run: |
-            set -eo pipefail
-
-            # please keep the matrices sorted in ascending order by the following:
-            #
-            #     [ARCH, PY_VER, CUDA_VER, LINUX_VER]
-            #
-            export MATRIX="
-            # amd64
-            - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'amd64', PY_VER: '3.9',  CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'amd64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'amd64', PY_VER: '3.11', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04' }
-            # arm64
-            - { ARCH: 'arm64', PY_VER: '3.9',  CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'arm64', PY_VER: '3.9',  CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'arm64', PY_VER: '3.10', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'arm64', PY_VER: '3.11', CUDA_VER: '11.8.0', LINUX_VER: 'ubuntu22.04' }
-            - { ARCH: 'arm64', PY_VER: '3.11', CUDA_VER: '12.2.2', LINUX_VER: 'ubuntu22.04' }
-            "
-
-            MATRIX="$(
-              yq -n -o json 'env(MATRIX)' | \
-              jq -c '${{ inputs.matrix_filter }} | if (. | length) > 0 then {include: .} else "Error: Empty matrix\n" | halt_error(1) end'
-            )"
-
-            echo "MATRIX=${MATRIX}" | tee --append "${GITHUB_OUTPUT}"
     build:
-      needs: compute-matrix
       strategy:
         fail-fast: false
-        matrix: ${{ fromJSON(needs.compute-matrix.outputs.MATRIX) }}
-      runs-on: "linux-${{ matrix.ARCH }}-${{ inputs.node_type }}"
+      runs-on: linux-${{ inputs.CPU }}-gpu-${{ inputs.GPU }}-latest-1
       env:
         RAPIDS_ARTIFACTS_DIR: ${{ github.workspace }}/artifacts
       container:
-        image: rapidsai/ci-conda:cuda${{ matrix.CUDA_VER }}-${{ matrix.LINUX_VER }}-py${{ matrix.PY_VER }}
+        image: rapidsai/ci-conda:cuda12.2.2-ubuntu22.04-py3.10
```

Comment:
The conda test below more or less depends on the fact that this image uses python 3.10, and that it only tests python 3.10 builds. Which is fine for now, given that our current python tests also only tests 3.10.